package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@globality/nodule-express",
-    "version": "0.4.0",
+    "version": "0.4.1",
```

Comment:
I would say this is 0.5.0 as this is a new feature and not a bug fix

Reply:
New PR -> https://github.com/globality-corp/nodule-express/pull/48